packages/mdc-menu/foundation.js

```diff
@@ -297,7 +322,12 @@ class MDCMenuFoundation extends MDCFoundation {
     const isEscape = key === 'Escape' || keyCode === 27;
 
     if (isEnter || isSpace) {
-      this.handlePossibleSelected_(evt);
+      // If our menu didn't catch the keydown, then the user likely hit the
```

Comment:
There is no "we" or "our" in comments. The program is the thing running. Try and be specific about which part of the program is doing what action.

For example,

"If MDCMenu does not see a keydown event, then it assumes the user triggered an enter or space event"

Also, how is enter and space NOT a keydown event??